html/php-challenge/index.php

```diff
@@ -1,12 +1,12 @@
 <?php
 session_start();
-require('dbconnect.php');
+require('dbconnect.php'); //ログイン画面
```

Comment:
このコメントは意味が不明瞭なので消しましょう
```suggestion
require('dbconnect.php');
```